src/rocprof_compute_base.py

```diff
@@ -62,7 +66,7 @@
 }
 
 
-class Omniperf:
+class rocprof_Compute:
```

Comment:
Stylistic feedback, but why the weird casing here? Not a big deal, but let's make sure we're consistent with casing in class names throughout the project.

Reply:
Just explaining the rationale: I was following the original uppercase Omniperf  here (which was the only uppercase Omniperf), when the class is instantiated, the original code was `omniperf = Omniperf()`, and I kept this same format: `rocprof_compute = rocprof_Compute()`.

Reply:
Okay, thank you. If you don't feel strongly one way or the other, I recommend we change to class name to `Rocprof_Compute`. This adheres to Python's PEP 8 style:
https://peps.python.org/pep-0008/#class-names

Reply:
Maybe just RocprofCompute ?

Reply:
I'd stick with general Python conventions on this one; use Pascal case and forgo the underscore. So: `RocprofCompute` or `RocProfCompute`.